### PR TITLE
store object with id

### PR DESCRIPTION
**Pull request by christylaguardia**

### CODE REVIEW

File: tests/store.test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            name: 'pluto',
+            breed: 'hound'
+        });
+        store.save({
+            name: 'snoopy',
+            breed: 'beagle'
+        });
+        let fetchedDogs = store.getAll();
+        assert.equal(fetchedDogs.length, 4); // count of dogs includes the 2 dogs added in test above, 
+    });
```
Comment (nsmeds) [inline tests/store.test.js]: Per the [https://github.com/cfpdx-401JS-summer-2017/in-memory-object-store/blob/master/LAB.md](LAB.md), you should also test that getAll() returns an empty array when no objects.

File: tests/store.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    it('finds a dog by id and returns that dog object', () => {
+        let charlie = store.save({
+            name: 'scooby doo',
+            breed: 'great dane'
+        });
+        let foundDog = store.get(charlie._id);
+        assert.deepEqual(foundDog, charlie);
+    });
```
Comment (nsmeds) [inline tests/store.test.js]: Per the [https://github.com/cfpdx-401JS-summer-2017/in-memory-object-store/blob/master/LAB.md](LAB.md), you should also test that get(id) returns null if that id does not exist.
